geode-core/src/main/java/org/apache/geode/management/internal/cli/remote/CommandExecutor.java

```diff
@@ -130,17 +142,43 @@ protected Object invokeCommand(Object command, GfshParseResult parseResult) {
       return resultModel;
     }
 
+    String groupsToUpdate = "";
     String groupInput = parseResult.getParamValueAsString("group");
-    if (groupInput == null) {
-      groupInput = "cluster";
+    TabularResultModel table = null;
+    if (StringUtils.isBlank(groupInput)
```

Comment:
For better readability:
if(!StringUtils.isBlank){
// use the group
}
else if (command is instance of UpdateAllMaker){
// pass all groups
}
else{
// pass in "cluster"
}